site/content/contribute/desktop/_index.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Desktop App"
+date: 2018-01-02T10:36:34-05:00
+section: "contribute"
+---
+
+# Desktop App
+The Mattermost desktop app is an [Electron](https://electronjs.org/) wrapper around the <a href="/contribute/webapp">webapp</a> project. It lives in the [mattermost/desktop](https://github.com/mattermost/desktop) repository. The desktop app runs on Windows, Linux, and macOS.
```

Comment:
For future reference `<a href="/contribute/webapp">webapp</a>` should work fine as `[webapp](/contribute/webapp)`, hugo is smart enough to handle that even if the base URL isn't at the root

Also from your screenshot it looks like you haven't pulled master in a while, which might have caused some different behavior